src/periphery/interfaces/ISablierMerkleFactory.sol

```diff
@@ -135,4 +180,54 @@ interface ISablierMerkleFactory {
     )
         external
         returns (ISablierMerkleLT merkleLT);
+
+    /// @notice Resets the Sablier fee to default.
+    /// @dev Emits a {ResetSablierFee} event.
+    ///
+    /// Notes:
+    /// - The default fee will only be applied to the future campaigns.
+    ///
+    /// Requiurements:
+    /// - The caller must be the admin.
+    ///
+    /// @param campaignCreator The user for whom the fee is being reset for.
+    function resetSablierFeeByUser(address campaignCreator) external;
+
+    /// @notice Sets the custom Sablier fee for a campaign creator.
+    /// @dev Emits a {SetSablierFee} event.
+    ///
+    /// Notes:
+    /// - The new fee will only be applied to the future campaigns.
+    ///
+    /// Requiurements:
+    /// - The caller must be the admin.
+    ///
+    /// @param campaignCreator The user for whom the fee is being set.
+    /// @param fee The new fee to be set.
+    function setSablierFeeByUser(address campaignCreator, uint256 fee) external;
+
+    /// @notice Sets the default Sablier fee for claiming an airstream.
+    /// @dev Emits a {SetDefaultSablierFee} event.
+    ///
+    /// Notes:
+    /// - The new default fee will only be applied to the future campaigns.
+    ///
+    /// Requiurements:
+    /// - The caller must be the admin.
+    ///
+    /// @param defaultFee The new detault fee to be set.
+    function setDefaultSablierFee(uint256 defaultFee) external;
+
+    /// @notice Withdraws the Sablier fees accrued on `merkleLockup` to the provided address.
+    /// @dev Emits a {WithdrawSablierFees} event.
+    ///
+    /// Notes:
+    /// - This function transfers ETH to the provided address. If the receiver is a contract, it must be able to receive
+    /// ETH.
+    ///
+    /// Requirements:
+    /// - The caller must be the admin.
+    ///
+    /// @param to The address to receive the Sablier fees.
+    function withdrawFees(address payable to, ISablierMerkleBase merkleLockup) external;
```

Comment:
merkls instant isn't considered a merkle "lockup", so, we should rename the param to something else like `sablierMerkle`

Reply:
`merkleBase`? given the type is `ISablierMerkleBase`. merkle is a known keyword to represent merkle tree so calling it sablier merkle may not be appropriate.

Reply:
`markleBase` works as well for me